scalene/scalene_profiler.py

```diff
@@ -224,6 +231,8 @@ def parse_args() -> Tuple[argparse.Namespace, List[str]]:
             os.environ["LD_PRELOAD"] = os.path.join(
                 os.path.dirname(__file__), "libscalene.so"
             )
+            if arguments.malloc_replacement:
+                os.environ["LD_PRELOAD"] += ":" + arguments.malloc_replacement
```

Comment:
Need to also do this for OS X.

Reply:
this was removed in another commit, it was from my attempt to add in a replacement memory allocator

Reply:
gotcha